server/graphql/v2/interface/FileInfo.ts

```diff
@@ -26,7 +26,7 @@ export const fileInfoFields = {
   size: {
     type: new GraphQLNonNull(GraphQLInt),
     description: 'Size of the file in bytes',
-    resolve: file => file.fileSize,
+    resolve: file => file.fileSize || 0,
```

Comment:
Do we check this on the front-end (and not show the file size) or just show 0kb on the front-end after this fix?

Reply:
It will show 0. Not ideal, but it's an acceptable fix for now. We'll iterate with https://github.com/opencollective/opencollective-api/pull/8498 to fetch all file sizes from S3, ultimately they'll be all non-null